internal/controller/component/component_adapter.go

```diff
@@ -135,10 +135,9 @@ func (a *Adapter) createUpdatedSnapshot(snapshotComponents *[]applicationapiv1al
 	if snapshot.Labels == nil {
 		snapshot.Labels = map[string]string{}
 	}
-	snapshotType := gitops.SnapshotCompositeType
-	if len(*snapshotComponents) == 1 {
-		snapshotType = gitops.SnapshotComponentType
-	}
+
+	snapshotType := gitops.SnapshotComponentType
```

Comment:
I don't think this would work because the new Snapshot will not have a component label. Maybe we can not have a label at all for now?